code/modules/projectiles/projectile/beams.dm

```diff
@@ -179,7 +179,7 @@
 			distance = 0
 		if(distance <= range)
 			var/mob/living/M = T
-			if (M.mob_size <= 2 && (M.find_type() & TYPE_ORGANIC))
+			if ((ismob(M)) && M.mob_size <= 2 && (M.find_type() & TYPE_ORGANIC))

```

Comment:
`istype(M)` instead of `ismob(M)`. Should be faster, and makes more sense since M is already established as a specific type.